README.md

```diff
@@ -43,7 +43,7 @@ The **RAG Experiment Accelerator** is config driven and offers a rich set of fea
 
 ## Installation
 
-Two options are available, install locally or inside a development container.
+Two options are available, install locally or run a  development container.
```

Comment:
Not 100% sure what this sentence is saying.
Is this better? If we're changing this, we should change the title below too.

```suggestion
Two options are available: install locally or run within development container.
```